generators/entity-client/files.js

```diff
@@ -40,6 +40,17 @@ const CLIENT_REACT_TEMPLATES_DIR = 'react';
 const angularFiles = {
     client: [
         {
+            path: ANGULAR_DIR,
+            templates: [
+                {
+                    file: 'entities/entity.model.ts',
+                    // using entityModelFileName so that there is no conflict when genertaing microservice entities
+                    renameTo: generator => `shared/model/${generator.entityModelFileName}.model.ts`
+                }
+            ]
+        },
+        {
+            condition: generator => !generator.embedded,
```

Comment:
What exactly does this mean that a generator is embedded? I ask because I see the embedded attribute of the generator object and I don't understand.

Reply:
I must admit that I don't understand it everything either, I followed readOnly PR (https://github.com/jhipster/generator-jhipster/pull/10561). It seems here that the generator contains current entity data (entityFolderName, entityFileName, readonly, ...).
So yes here embedded means that the current entity is embedded.

Reply:
This is correct. It's done this way currently.
But it should be renamed to isEmbedded too.
Actually you should use isEmbedded everywhere.

Reply:
I don't really like using the keyword `is` in variables, it should be used only on methods. If I rename this to `isEmbedded`, `readOnly` should also be renamed to `isReadOnly`, otherwise it won't be consistent! same thing for `jpaMetamodelFiltering`, `validation` ...

Reply:
@mshima I don't agree. If it's a function, then yes it should be called `isXXX`. If it's a variable, the name prevails.

Reply:
No problem